pax-macro/src/lib.rs

```diff
@@ -557,3 +600,100 @@ fn generate_include(name: &Ident, path: &PathBuf) -> TokenStream {
         const #const_name: &'static str = include_str!(#path_str);
     }
 }
+#[proc_macro_attribute]
+pub fn helpers(
+    _attr: proc_macro::TokenStream,
+    item: proc_macro::TokenStream,
+) -> proc_macro::TokenStream {
+    let input = parse_macro_input!(item as ItemImpl);
+    let struct_name = &input.self_ty;
+
+    let mut register_functions = vec![];
+
+    for item in input.items.iter() {
+        if let ImplItem::Method(method) = item {
+            let func_name = &method.sig.ident;
+
+            // Check if the function is static (doesn't use `self`)
```

Comment:
I believe the lingo is "associated function" in Rust — https://doc.rust-lang.org/reference/items/associated-items.html#associated-functions-and-methods (applies to error message below, too)

Reply:
good catch! updated!